src/transformers/models/perceiver/convert_perceiver_haiku_to_pytorch.py

```diff
@@ -0,0 +1,476 @@
+# coding=utf-8
+# Copyright 2021 The HuggingFace Inc. team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Convert Perceiver checkpoints originally implemented in Haiku."""
+
+
+import argparse
+import json
+import pickle
+from pathlib import Path
+
+import numpy as np
+import torch
+from PIL import Image
+
+import haiku as hk
+import requests
+from huggingface_hub import cached_download, hf_hub_url
+from transformers import (
+    PerceiverConfig,
+    PerceiverFeatureExtractor,
+    PerceiverForImageClassification,
+    PerceiverForImageClassificationConvProcessing,
+    PerceiverForImageClassificationFourier,
+    PerceiverForMaskedLM,
+    PerceiverForMultimodalAutoencoding,
+    PerceiverForOpticalFlow,
+    PerceiverTokenizer,
+)
+from transformers.utils import logging
+
+
+logging.set_verbosity_info()
+logger = logging.get_logger(__name__)
+
+# We will verify our results on an image of a dog
+
+
```

Comment:
```suggestion
```
(Move the comment after the function name if black reintroduces the two blank lines)